### ISSUE TITLE
Compilation instructions with OpenJDK

### ISSUE THREAD

**Issue by TechnologyClassroom** [mikeberger/borg_calendar#93]
Does this work with OpenJDK?

BORGCalendar/install/buildjre.sh needs `/bin/jlink`.  I do not see that packaged for Ubuntu.  https://packages.ubuntu.com/search?keywords=jlink&searchon=names&suite=focal&section=all

**Comment by mikeberger** [comment]
For developing on both windows and linux, I download the version of openjdk that I want/need from here: https://openjdk.java.net/ and just manually set JAVA_HOME and PATH to point to it. My master branch is using version 13. openjdk is up to 15 already?

jlink is used to build a custom JRE that is packaged with BORG. As per ORACLE, java programs should now package a JRE and not expect a JRE to be installed on the user's machine. This means the user doesn't have to worry about Java.

Given the new release frequency of Java, whatever openjdk is packaged with ubuntu is probably not the right version.

tl;dr
ORACLE has made an unholy mess out of desktop Java and I'm glad that I'm retired from Java software development.
